cameraCalib.py

```diff
@@ -9,12 +9,13 @@
 import sys
 import random
 
-# Define the chess board rows and columns
+# Define the chess board rows and columns, and length of the square side
 rows = 8
 cols = 6
+square_length_mm = 30.
 
 # Set the termination criteria for the corner sub-pixel algorithm
-criteria = (cv2.TERM_CRITERIA_MAX_ITER + cv2.TERM_CRITERIA_EPS, 30, 0.001)
+criteria = (cv2.TERM_CRITERIA_MAX_ITER + cv2.TERM_CRITERIA_EPS, square_length_mm, 0.001)
```

Comment:
The second value in `criteria` is not referring to the length of the square sides, but rather to the `maxCount`, i.e. the maximum number of iterations or elements to compute before termination.